tests/science/science.cfg

```diff
@@ -1 +1,21 @@
-[test]
+[science_test_snotel]
+driver = classic
+test_description = snotel point simulations
+global_parameter_file = global_param.classic.5.0_snotel.txt
+expected_retval = 0
+check = not_implemented
```

Comment:
Alright - so the `complete` check was failing because the output files have different start and end times depending on the length of the forcing files (differing forcing time series lengths between Fluxnet sites --> different output file time series start and end dates). The `output_file_nans` check was failing due to something weird going in in the `read_vic_ascii` function, which is now fixed.